server.py

```diff
@@ -185,6 +185,22 @@ def do_get_response(mongo_id):
 
     return jsonify({}), 404
 
+
+@app.route('/queue', methods=['POST'])
+def do_queue():
+    mongo_id = request.get_data().decode('UTF8')
```

Comment:
I think this endpoint should accept JSON to be consistent with the rest of the API, something like `{ id: "<mongo-id> }`